java/src/AdventDay1/input.txt

```diff
@@ -0,0 +1,2000 @@
+183
+185
+186
+182
```

Comment:
Same here. The input file is already in `challenges/day1/data/input.txt`. You should reference this one.